charts/spark-operator-chart/templates/controller/_helpers.tpl

```diff
@@ -97,6 +97,14 @@ Create the name of the pod disruption budget to be used by controller
 {{ include "spark-operator.controller.name" . }}-pdb
 {{- end -}}
 
+{{/*
+Create the name of the service used by controller for accessing pprof
+*/}}
+{{- define "spark-operator.controller.pprofServiceName" -}}
+{{ include "spark-operator.controller.name" . }}-pprof
+{{- end -}}
+
+
```

Comment:
```suggestion
{{/*
Create the name of the service used by controller
*/}}
{{- define "spark-operator.controller.serviceName" -}}
{{ include "spark-operator.controller.name" . }}-svc
{{- end -}}

```

What do you think about renaming the service for controller, so it can be used to expose metric, pprof and other endpoints?

Reply:
That's actually how we implemented it on our fork

Since lots of things have changed in the repo with the controller-runtime pr I did the minimum effort solution

I'll rename the file to service.yaml and accept your change